helm/charts/td-agent-barito/templates/tdagentbaritoconfig.yaml

```diff
@@ -75,6 +79,6 @@ data:
     </match>
     {{- end }}
 
-    {{- range .Values.extraConfig }}
-    {{ .config | nindent 4 }}
+    {{- range .Values.postMatchConfig }}
```

Comment:
This will break for the users who are already using `extraConfig` but makes sense. Lets add this to the changeLog as well as breaking changes in the next version.

Reply:
I've bumped the version and added this to changelog, marking as this as resolved